vercel.json

```diff
@@ -1438,6 +1438,62 @@
     {
       "source": "/docs/guides/migrate/deployment/deploy-database-changes-with-prisma-migrate",
       "destination": "/docs/guides/deployment/deploy-database-changes-with-prisma-migrate"
+    },
+    {
+      "source": "/docs/guides/deployment/deployment",
+      "destination": "/docs/guides/deployment/deploying-prisma"
+    },
+    {
+      "source": "/docs/guides/deployment/deployment-guides/deploying-to-heroku",
+      "destination": "/docs/guides/deployment/traditional-deployments/deploying-to-heroku"
```

Comment:
Another possibility instead of "traditional" would be to highlight that they are "long running" (vs. serverless or edge, that only run for a short while). But also not suuuuuper clean.

I wonder how others solve this problem.